com.wso2telco.dep.gw.ussdapi/ussdapigw-synapse/src/main/synapse-config/sequences/ussdInboundResponseSeq.xml

```diff
@@ -2,14 +2,19 @@
 <sequence name="ussdInboundResponseSeq" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
     <class name="com.wso2telco.dep.mediator.impl.ussd.USSDInboundResponseMediator"/>
     <property name="outboundUSSDMessage" expression="json-eval($.outboundUSSDMessageRequest.outboundUSSDMessage)" />
-    <class name="org.wso2telco.dep.nashornmediator.NashornMediator">
-        <property name="script" value='
-            var payload = mc.getPayloadJSON();
-            payload.outboundUSSDMessageRequest.responseRequest.notifyURL = mc.getProperty("subsEndPoint");
-            var str = mc.getProperty("outboundUSSDMessage");
-            var text = str.replace("\n", "\\n").replace("\r", "\\r");
-            payload.outboundUSSDMessageRequest.outboundUSSDMessage = text;
-            mc.setPayloadJSON(payload);
-        '/>
+    <property expression="fn:replace(fn:replace(syn:get-property('outboundUSSDMessage'), '\n', '\\n'), '\r', '\\r')" name="propertyReplacement" scope="default" type="STRING" xmlns:fn="http://www.w3.org/2005/xpath-functions"/>
+
+    <class name="com.wso2telco.dep.common.mediation.UtilMediator">
```

Comment:
As I see there are so many code duplications while calling the UtilMediator. It's better to implement one common sequence to Single Property Replacement and call it from other sequences where propertyName,propertyPath, msgContextProperty properties keep as dynamic values.